### ISSUE TITLE
Visual Studio incorrectly pauses on conditional breakpoints.

### ISSUE THREAD

**Issue by dragon3025** [godotengine/godot-vscode-plugin#267]
I'm using visual studio code with godot-tools extension. To test this out, I set a conditional breakpoint with a condition that's impossible to achieve.

The game incorrectly freezes on the breakpoint, and from what I seen in videos, there's supposed to be a yellow marker outline on the breakpoint, but that's not the case. My only options when it freezes is pause, restart, or start which is a sign that it hasn't actually hit the breakpoint.

When I disable the breakpoint the game runs fine, so it's not the coding.

EDIT: I changed the title, because the freezing isn't happening anymore, but the incorrect pausing is.

**Comment by Calinou** [comment]
For reference, this was originally posted on https://godotforums.org/discussion/26223/when-using-visual-studio-code-for-debugging-it-incorrectly-freezes-on-breakpoints#latest.

@dragon3025 Which Godot version are you using?

**Comment by dragon3025** [comment]
My Godot version is: v3.3.stable.official
My Visual studio version is 1.56.0

I've attached a reduced version of my project.

Before I reduced it I tested it some more. I didn't run into the problem again, except until after a few tries. I don't know what I did to cause it, but it could be after an error in a breakpoint (it did it again after I fixed the error), or it could be after I debugged it in godot instead of vs.

Before and after I reduced it, it was breaking on the breakpoint incorrectly (this time actually breaking on instead of just freezing the game).

[Action RPG.zip](https://github.com/godotengine/godot-vscode-plugin/files/6446613/Action.RPG.zip)
